### PR TITLE
[dev-env] validate client code - look for specific subdirectories

### PR DESCRIPTION
**Pull request by pschoffer**
## Description

To prevent folks from linking to an incorrect site-code folder we added basic validation that there is `themes` subdirectory at least.

## Steps to Test

```
npm run build && ./dist/bin/vip-dev-env-create.js --slug test --client-code="/tmp"
```

Or type something like that during wizard.

### CODE REVIEW

**Comment by rinatkhaziev** [comment]
I think we should match the validation in production and list all that is missing. 
For example, I pointed to a copy of my repo that only has a themes folder, I mistakenly point to that folder, and VIP-CLI deems this valid. I spin up. the environment only to realize the site is broken. I get annoyed and open a ticket.

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
It's ok :)

**Comment by pschoffer** [comment]
Done I added the same as the prod check does.

File: src/lib/dev-environment/dev-environment-cli.js [hunk 1]
```diff
@@ -253,6 +251,47 @@ async function processComponent( component: string, preselectedValue: string, de
...
+
+		const missingFiles = [];
+		for ( const file of files ) {
+			const filePath = path.resolve( providedPath, file );
+			if ( ! fs.existsSync( filePath ) ) {
+				missingFiles.push( file );
+			}
+		}
+		if ( missingFiles.length > 0 ) {
+			const message = `Provided path "${ providedPath }" is missing following files/folders: ${ missingFiles.join( ', ' ) }`;
```
Comment (rinatkhaziev) [inline src/lib/dev-environment/dev-environment-cli.js]:
```suggestion
			const message = `Provided path "${ providedPath }" is missing following files/folders: ${ missingFiles.join( ', ' ) }. Learn more: https://docs.wpvip.com/technical-references/vip-codebase/#1-wordpress`;
```

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**